__tests__/app.test.js

```diff
@@ -4,7 +4,7 @@ const app = require('../app');
 const db = require('../db/data/test-data/index');
 const seed = require('../db/seeds/seed');
 const testData = require('../db/data/test-data');
-
+jest.setTimeout(10000000);
```

Comment:
Can I ask why you needed to include this? It's quite a long time

Reply:
so i ran into an issue,  jest kept timing out.I eventually found out something was actually wrong with my code. Should take it out.